saga/sagalogs/memory.go

```diff
@@ -0,0 +1,155 @@
+// Package sagalogs provides implementations of SagaLog.
```

Comment:
This was actually copied from in_memory_saga_log.go with only moderate modification, but github not handling that well.

Reply:
just a thought.... in the spirit of extreme programming's don't build it till you are going to use it....  should we just dump saga log completely - are we using it for anything meaningful?

Reply:
I believe we still need it for keeping track of tasks at all currently. I'm not against ditching it but that's way beyond scope. We may have to look into that closer for another internal issue ("reduce scheduler queueing latency by a lot").